### ISSUE TITLE
No commodore on Spigot?

### ISSUE THREAD

**Issue by SlimeDog** [InsightsPlugin/Insights#175]
Spigot 1.19.2-3566
Insights 6.12.1 (from SpigotMC)

On server start-up:
```
[15:51:30] [Server thread/INFO]: [Insights] Enabling Insights v6.12.1
[15:51:31] [Server thread/INFO]: [Insights] Loaded limit 'bed-limit.yml'
[15:51:31] [Server thread/INFO]: [Insights] Loaded limit 'redstone-limit.yml'
[15:51:31] [Server thread/INFO]: [Insights] Loaded limit 'tile-limit.yml'
[15:51:31] [Server thread/ERROR]: Error occurred while enabling Insights v6.12.1 (Is it up to date?)
dev.frankheijden.insights.dependencies.cloud.bukkit.BukkitCommandManager$BrigadierFailureException: Could not initialize Brigadier mappings. Reason: commodore not present (Could not initialize Brigadier mappings. Reason: commodore not present ())
        at dev.frankheijden.insights.dependencies.cloud.bukkit.BukkitCommandManager.registerBrigadier(BukkitCommandManager.java:329) ~[?:?]
        at dev.frankheijden.insights.dependencies.cloud.paper.PaperCommandManager.registerBrigadier(PaperCommandManager.java:125) ~[?:?]
        at dev.frankheijden.insights.Insights.loadCommands(Insights.java:278) ~[?:?]
        at dev.frankheijden.insights.Insights.onEnable(Insights.java:136) ~[?:?]
        at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:264) ~[spigot-api-1.19.2-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:342) ~[spigot-api-1.19.2-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:480) ~[spigot-api-1.19.2-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.craftbukkit.v1_19_R1.CraftServer.enablePlugin(CraftServer.java:525) ~[spigot-1.19.2-R0.1-SNAPSHOT.jar:3566-Spigot-584b459-c379a6b]
        at org.bukkit.craftbukkit.v1_19_R1.CraftServer.enablePlugins(CraftServer.java:439) ~[spigot-1.19.2-R0.1-SNAPSHOT.jar:3566-Spigot-584b459-c379a6b]
        at net.minecraft.server.MinecraftServer.loadWorld0(MinecraftServer.java:584) ~[spigot-1.19.2-R0.1-SNAPSHOT.jar:3566-Spigot-584b459-c379a6b]
        at net.minecraft.server.MinecraftServer.loadLevel(MinecraftServer.java:403) ~[spigot-1.19.2-R0.1-SNAPSHOT.jar:3566-Spigot-584b459-c379a6b]
        at net.minecraft.server.dedicated.DedicatedServer.e(DedicatedServer.java:255) ~[spigot-1.19.2-R0.1-SNAPSHOT.jar:3566-Spigot-584b459-c379a6b]
        at net.minecraft.server.MinecraftServer.v(MinecraftServer.java:968) ~[spigot-1.19.2-R0.1-SNAPSHOT.jar:3566-Spigot-584b459-c379a6b]
        at net.minecraft.server.MinecraftServer.lambda$0(MinecraftServer.java:293) ~[spigot-1.19.2-R0.1-SNAPSHOT.jar:3566-Spigot-584b459-c379a6b]
        at java.lang.Thread.run(Thread.java:833) [?:?]
Caused by: dev.frankheijden.insights.dependencies.cloud.bukkit.BukkitCommandManager$BrigadierFailureException: Could not initialize Brigadier mappings. Reason: commodore not present ()
        at dev.frankheijden.insights.dependencies.cloud.bukkit.CloudCommodoreManager.<init>(CloudCommodoreManager.java:55) ~[?:?]
        at dev.frankheijden.insights.dependencies.cloud.bukkit.BukkitCommandManager.registerBrigadier(BukkitCommandManager.java:324) ~[?:?]
        ... 14 more
```
Works fine on Paper 1.19.2-117.

**Comment by FrankHeijden** [comment]
Thanks, will be fixed in v6.12.2

**Comment by SlimeDog** [comment]
Confirmed fixed in Insights 6.12.2. Thanks.
